spec/css-spec.coffee

```diff
@@ -126,9 +437,9 @@ describe 'CSS grammar', ->
         expect(tokens[0]).toEqual value: '.', scopes: ['source.css', 'meta.selector.css', 'entity.other.attribute-name.class.css', 'punctuation.definition.entity.css']
         expect(tokens[1]).toEqual value: '_', scopes: ['source.css', 'meta.selector.css', 'entity.other.attribute-name.class.css']
 
-      it 'does not tokenize tokens containing ASCII punctuation or symbols other than "-" and "_"', ->
+      it 'does not tokenize tokens containing unescaped ASCII punctuation or symbols other than "-" and "_"', ->
         {tokens} = grammar.tokenizeLine '.B&W'
-        expect(tokens[0]).toEqual value: '.B&W', scopes: ['source.css', 'meta.selector.css']
+        expect(tokens[0].scopes).not.toEqual ['source.css', 'meta.selector.css', 'entity.other.attribute-name.class.css', 'punctuation.definition.entity.css']
```

Comment:
Can we avoid `.not` please?  Those work under the assumption that the scopes will never change, which as we can see in this PR definitely isn't true.

Yes, testing the actual scopes is more ugly but ensures that this spec doesn't become useless.